app/experimenter/nimbus-ui/src/components/PreviewURL/index.stories.tsx

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import PreviewURL from ".";
+import { getStatus } from "../../lib/experiment";
+import { mockExperiment } from "../../lib/mocks";
+
+export default {
+  title: "components/PreviewURL",
+  component: PreviewURL,
+};
+
+export const Basic = () => {
+  const experiment = mockExperiment();
+  const status = getStatus(experiment);
+
+  return <PreviewURL {...{ ...experiment, status }} />;
```

Comment:
Could use a little padding here as a story - [I think we've used an `mr-3` elsewhere](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/components/LinkExternal/index.stories.tsx#L11)?

Not a biggie, though

Reply:
Deal! We should make a "padding" decorator